app/config/frontendAppConfig.scala

```diff
@@ -154,4 +157,7 @@ class FrontendAppConfig @Inject()(val runModeConfiguration: Configuration, envir
   override val agentServicesUrl: String = agentServicesHost + getString(ConfigKeys.agentServicesUrl)
 
   override lazy val staticDateValue: String = getString(Keys.staticDateValue)
-}
+
+  override lazy val signUpServiceHost: String = getString(Keys.signUpServiceHost)
+  override lazy val signUpServiceUrl: String => String = vatNumber =>  getString(Keys.signUpServiceUrl) + s"$vatNumber"
```

Comment:
hi can you just update this with having the host at the beginning also ,then make a PR for app-config-base to set the host to ""